### ISSUE TITLE
[Screen Reader-Help-Projects]Narrator does not give information when the focus is on red icons in Scan mode.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2583]
**User Experience:**
User who depends on Screen reader will get confused if narrator does not give information when the focus in on red icons in Scan mode.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Screen reader: Narrator

**Repro Steps:-**
1- Navigate to https://makecode.microbit.org/acc
2-Navigate to the Microbit section element and select Code control given on it.
3-Navigate to the Help control lying in the header section on the page and select it.
4-Navigate to 'Projects' control and select it.
5. Select step "Smiley buttons"
6- Verify Screen Reader behavior.

**Actual Result:-**
In scan mode(CAPS + Space key), Narrator focus is not visible on red icons and reading them as images.

**Expected Result:-**
In scan mode(CAPS + Space key), Narrator focus is not visible on red icons and reading them as images.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b24201d1-2276-4471-8183-e1e5353031ad}

**Note:** Not able to check with NVDA due to inconsistent behavior.
Narrator does not give information when the focus is on red icons in items mode(CAPS + Arrow keys)

Same issue exist in scenario:
1. Help-->References-->Narrator reads multiple times images in Scan mode. Please refer attachment for ore in details.
2. Help-->References-->Select any section-->Narrator reads multiple times images in Scan  mode. 
3. Help-->Blocks-->Narrator reads multiple times images in Scan mode. 
4. Help-->Blocks-->Select any option-->Narrator reads multiple times images in Scan mode. 

Please refer the attachment
[MAS4.2.10_Projects_Narrator focus not visible not provide info.zip](https://github.com/Microsoft/pxt/files/1196425/MAS4.2.10_Projects_Narrator.focus.not.visible.not.provide.info.zip)

**Comment by veler** [comment]
This is part of Blocky, a third-part component.

**Comment by abchatra** [comment]
We have got exception for Blockly bugs from Julia Liuson (CVP). Closing these as Won't Fix. Following is the reasoning.

----------------------

MakeCode can be used in either Text editor or Block editor.

Blocks editor is inherently visual and is based on google blockly. Usage requires drag and drop functionality and we don’t have a equivalence in MakeCode for this w.r.t accessibility. 
We have evaluated couple of research projects which tries to find a equivalence to block based coding for accessibility needs: 
•	Project Torino (Requires students to buy physical blocks)
•	Accessible blockly (This is text based coding without syntax errors)
Both these are drastic departure from the blockly and provides a complete new experience. These are still in research and we intend not to use them in MakeCode at this point of time.

As workaround, the text editor in MakeCode is the way we see students with the need of accessibility using MakeCode.
